app/task7.cs

```diff
@@ -0,0 +1,33 @@
+namespace task7
+{
+    public class PointInCircle
+    {
+        public static void Result()
+        {
+            double x;
+            double y;
+
+            if (!double.TryParse(Console.ReadLine(), out x)
+                || !double.TryParse(Console.ReadLine(), out y))
+            {
+                Console.WriteLine("Incorrect input");
+                return;
+            }
+
+            const double RADIUS = 2;
+            double result = Math.Sqrt(Math.Pow(x, 2) + Math.Pow(y, 2));
```

Comment:
Extract calculation in a function. I think you have the same calculation before. You can have utils file or commonFunctions file?